lib/Zonemaster/Backend/DB.pm

```diff
@@ -130,6 +133,30 @@ sub process_unfinished_tests {
     }
 }
 
+sub generate_deterministic_hash {
```

Comment:
1. Now that a new method is added to this module, it should also be documented with POD. Please compare with e.g. Config.pm
2. The name of the method is misleading. It actually does two things:
   1. Normalize the parameter hash
   2. Create a finger print (hash) of the normalized parameter hash

The two operations do not need to coexist in the same method, and I suggest that the method is broken up into two methods, which will make it easier to understand the code.

Reply:
> It actually does two things

Good point, I'll review my approach